dwst/scripts/particles.js

```diff
@@ -238,6 +261,20 @@ export function parseParticles(particleString) {
   return parsedParticles;
 }
 
+export function parseParticles(particleString) {
+  let parsedParticles;
+  try {
+    parsedParticles = parseParticles2(particleString);
```

Comment:
I'm wondering if eslint would complain if we did `return` inside `try` here. It would let us get rid of `let`.

Reply:
eslint complains not after changing to that. Could we also just directly ```return parsedParticles2(particleString);```?

Reply:
Our tests + eslint are probably able to tell us if that doesn't work.